amdgpukmsfw/Makefile

```diff
@@ -39,37 +42,45 @@ _VALID_AMDGPUKMODS=	aldebaran \
 			psp_13_0_0 \
 			psp_13_0_4 \
 			psp_13_0_5 \
+			psp_13_0_6 \
 			psp_13_0_7 \
 			psp_13_0_8 \
 			psp_13_0_10 \
 			psp_13_0_11 \
+			psp_14_0_0 \
 			raven \
 			raven2 \
 			renoir \
+			sdma_4_4_2 \
 			sdma_5_2_6 \
 			sdma_5_2_7 \
 			sdma_6_0_0 \
 			sdma_6_0_1 \
 			sdma_6_0_2 \
 			sdma_6_0_3 \
-			sdma_6_0_4 \
```

Comment:
Isn't this used by some drivers ?

Reply:
The firmware does not exist at all in this repository or upstream :-)

Reply:
Ok!